### PR TITLE
fixes zoom level color bug

### LINKED ISSUE #1056: Zoom level bug
IMG

### PR DESCRIPTION
**Pull request by waahm7**
Fixes #1056 

Changes: 
1-Instead of getting Nightmode from SharedPreference, used SharedPreferenceUtil nightMode() function, So that it correctly handles nightMode value even when Auto Night Mode is enabled. 

Screenshots/GIF for the change: [If possible, please add relevant screenshots/GIF]
54524973_2093356744114906_2636128657155817472_n
53861050_3302015163206422_1108615876111761408_n

### CODE REVIEW

**Comment by waahm7** [comment]
@abdulwd @mhutti1 Please review this!

**Comment by prajurock** [comment]
@waahm7 that not the correct pr for this issue.And i request you to comment before taking any issue.I am currently working on this.

**Comment by iadeelzafar** [comment]
@waahm7 You should wait for the response of the other person and if he confirms that he's not working, that's when you should start working on the PR.

**Comment by waahm7** [comment]
@prajurock Okay, but why is it not correct?

**Comment by prajurock** [comment]
Because the issue is zoom level button show be grayed when zoom control switch is on.But in automate night mode it showing error.

**Comment by waahm7** [comment]
@prajurock I am sorry for not waiting for the reply but this PR correctly solves the issue that is wrong color scehemes whenever night mode button is enabled but it is day theme due to automated night mode being on or vice versa. It should be grayed when zoom controls is off, which is already correct. Can I reopen the PR so that it can be accepted?

**Comment by prajurock** [comment]
Sorry, I was busy and checked now. Yes, this PR is okay but wait for a reply. You can open this PR.

**Comment by waahm7** [comment]
@prajurock  Okay, thanks!

**Comment by waahm7** [comment]
@abdulwd @mhutti1 please review this!

File: app/src/main/java/org/kiwix/kiwixmobile/settings/SliderPreference.java [hunk 1]
```diff
@@ -171,8 +169,10 @@ protected void onBindView(View view) {
   }
 
   private void setGrayState(View view) {
-    boolean enabled = getPreferenceManager().getSharedPreferences().getBoolean(PREF_ZOOM_ENABLED, false);
-    boolean Nightmode = getPreferenceManager().getSharedPreferences().getBoolean(PREF_NIGHTMODE, false);
+    SharedPreferenceUtil sharedPreferenceUtil = new SharedPreferenceUtil(getContext());
+    boolean enabled = sharedPreferenceUtil.getPrefZoomEnabled();
+    boolean Nightmode = sharedPreferenceUtil.nightMode();
```
Comment (abdulwd) [inline app/src/main/java/org/kiwix/kiwixmobile/settings/SliderPreference.java]: The variable name should begin with a lowercase. If you want you can clean up this file by removing the Hungarian notations wherever used and the lint warnings.

**Comment by prajurock** [comment]
You should remove this blank line. The best way to avoid these type of mistakes is to import Square's code style as described in CODE_STYLE.md and reformat the code before committing using Ctrl + Shift + L or from Code -> Reformat Code but the easiest way is to go to menu select VCS -> Git -> Commit File and select the files to be committed and make the top 3 options on the right side checked (make sure to import Square's code style before doing this otherwise Android's default code style will be used):

**Comment by waahm7** [comment]
Sorry for the delay. I was busy due to my exams.

**Comment by waahm7** [comment]
@prajurock what blank line ?

**Pull request merged**